storage/Devices/BcacheImpl.cc

```diff
@@ -53,17 +54,29 @@ namespace storage
 
     const char* DeviceTraits<Bcache>::classname = "Bcache";
 
+    const vector<string> EnumTraits<CacheMode>::names({
+	"writethrough", "writeback", "writearound", "none"
+    });
 
     Bcache::Impl::Impl(const string& name)
-	: Partitionable::Impl(name)
+	: Partitionable::Impl(name), cache_mode(CacheMode::WRITETHROUGH),
+          sequential_cutoff(8 * MiB), writeback_percent(10)
     {
 	update_sysfs_name_and_path();
     }
 
 
     Bcache::Impl::Impl(const xmlNode* node)
-	: Partitionable::Impl(node)
+	: Partitionable::Impl(node), cache_mode(CacheMode::WRITETHROUGH),
+          sequential_cutoff(8 * MiB), writeback_percent(10)
     {
+        string tmp;
+
+        cache_mode = CacheMode::WRITETHROUGH;
+        if (getChildValue(node, "cache_mode", tmp))
+            cache_mode = toValueWithFallback(tmp, CacheMode::WRITETHROUGH);
+	getChildValue(node, "writeback_percent", writeback_percent);
+	getChildValue(node, "sequential_cutoff", sequential_cutoff);
```

Comment:
misaligned

(Which editor do you use? I collect tips to configure editors to avoid these annoyances.)

Reply:
to be honest, I prefer if there is e.g. clang format configuration, so I can check it miself. I think reason why it is misaligned is that libstorage-ng use ancient yast style with mixture of tabs and spaces, which is not so easy to configure in editor (at least I never succeed ).

Reply:
Please see https://github.com/openSUSE/libstorage-ng/blob/master/.editorconfig

Reply:
Yes, that .editorconfig seems to properly represent doc/style-guide.md.

Concerning clang format the result I had so far were always unsatisfactory.

Reply:
Fair point, EditorConfig addresses the "encoding" and "implementation" of "indents" but it does not help you with specifying where the indents should go (or braces, for that matter).

Reply:
@mvidner but thanks for it, as it helps for lines where it does not use tabs properly and now new lines looks properly indented. So using it carefully to parts of file only and for new lines.